src/sentry/models/groupowner.py

```diff
@@ -57,3 +57,18 @@ def owner(self):
         from sentry.api.fields.actor import Actor
 
         return Actor.from_actor_identifier(self.owner_id())
+
+
+def get_owner_details(group_list):
+    group_ids = [g.id for g in group_list]
+    group_owners = GroupOwner.objects.filter(group__in=group_ids)
+    owner_details = {
+        go.group_id: {
+            "type": go.type,
```

Comment:
Should we convert the type back to the string name? We often convert integer enums back to string values like in transaction.status, activity types, group subscription reasons and more. The string values are simpler for end users as they don't need to refer to an enum to know what the current value is.

Reply:
Will do, thanks!